test/dpul_collections_web/live/browse_live_test.exs

```diff
@@ -80,4 +80,33 @@ defmodule DpulCollectionsWeb.BrowseLiveTest do
     view
     |> has_element?(".item-link")
   end
+
+  test "renders primary thumbnail only  when there is only one image url", %{conn: conn} do
```

Comment:
Should this new test be added here, or maybe in a different PR?

Reply:
If it's not here this PR fails because of coverage.

Reply:
aha, thanks